### PR TITLE
New package: ncdns-0.0.2

### PR DESCRIPTION
**Pull request by ananteris**

### CODE REVIEW

File: srcpkgs/ncdns/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'ncdns'
+pkgname=ncdns
+version=0.0.2
+revision=1
+only_for_archs="x86_64-musl x86_64 i686"
+build_style=go
+hostmakedepends="git libcap-devel"
```
Comment (Duncaen) [inline srcpkgs/ncdns/template]: Move `libcap-devel` to `makedepends`.

File: srcpkgs/ncdns/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+go_get=yes
```
Comment (Duncaen) [inline srcpkgs/ncdns/template]: You already provide a distfile, is `go get` really needed?
If not I guess you can drop the `git` `hostmakedepends` too.

File: srcpkgs/ncdns/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+go_import_path="github.com/hlandau/ncdns"
+short_desc="Nameserver for Namecoin .bit domains"
+maintainer="ananteris <ananteris@mailinator.com>"
+license="GPL-3"
+homepage="https://github.com/hlandau/ncdns"
+distfiles="https://github.com/hlandau/${pkgname}/archive/v${version}.tar.gz"
+checksum="a9457216f5bbc3c3b41e0538b78f8f67ad9a04304743a352a7325eef6af4a439"
+
+do_install() {
+	vbin ${GOPATH}/bin/ncdns
```
Comment (Duncaen) [inline srcpkgs/ncdns/template]: The `go` `build_style` already searches and installs binaries in `$GOPATH` by default (https://github.com/voidlinux/void-packages/blob/master/common/build-style/go.sh#L17).
You can remove it here, and rename the function to `post_install` to just install the config file.
I think if you drop `go_get=yes` you will be in the `ncdns/` directory and you dont need the long path.

Comment (ananteris) [inline srcpkgs/ncdns/template]: All of these have been resolved.

**Pull request merged**